data/tabs.json

```diff
@@ -7,5 +7,10 @@
     "id": "impl",
     "title": "impl period",
     "description": "Contribute to Rust's 2017 impl period"
+},
+{
+    "id": "blitz",
+    "title": "libz blitz",
+    "description": "Find issues in libz blitz evaluated crates"
```

Comment:
Could you explain a bit more about what the libs blitz is please? (This description doesn't have to be short, you can also use markdown)

Reply:
Ah no worries! I thought it was meant to be a sound-bite sized thing. I'll flesh this out.